ocaml/xapi/xapi_vgpu.ml

```diff
@@ -28,9 +51,11 @@ let create' ~__context  ~vM ~gPU_group ~device ~other_config ~_type ~powerstate_
     raise (Api_errors.Server_error (Api_errors.feature_restricted, []));
   if powerstate_check then
     Xapi_vm_lifecycle.assert_initial_power_state_is ~__context ~self:vM ~expected:`Halted;
-  if not(valid_device device) then
-    raise (Api_errors.Server_error (Api_errors.invalid_device, [device]));
-
+  let device_id =
+    try
```

Comment:
This `try..with` does nothing. The exception would have escaped without it - so why is it here?